.mailmap

```diff
@@ -248,9 +247,40 @@ Akshay Siramdas <akshaysiramdas@gmail.com> AkshaySiramdas <akshaysiramdas@gmail.
 Nitin Chaudhary <nitinmax1000@gmail.com> Nitin <nitinmax1000@gmail.com>
 Alex Argunov <sajkoooo@gmail.com> alex argunov <sajkoooo@gmail.com>
 Abhishek Garg <abhishekgarg119@gmail.com> abhishek garg <abhishekgarg119@gmail.com>
-Gaurav Dhingra <axyd0000@gmail.com> Gaurav Dhingra <igauravdhingra@protonmail.com>
+Gaurav Dhingra <gauravdhingra.gxyd@gmail.com> Gaurav Dhingra <igauravdhingra@protonmail.com>
+Gaurav Dhingra <gauravdhingra.gxyd@gmail.com> Gaurav Dhingra <axyd0000@gmail.com>
 Krit Karan <kritkaran.b13@iiits.in> Krit Karan <kritkaran94@users.noreply.github.com>
 Arihant Parsoya <parsoyaarihant@gmail.com> arihant parsoya <parsoyaarihant@gmail.com>
 Harsh Gupta <mail@hargup.in> Harsh Gupta <gupta.harsh96@gmail.com>
 Sumith <sumith1896@gmail.com> Sumith1896 <sumith1896@gmail.com>
 Kshitij Saraogi <KshitijSaraogi@gmail.com> kshitij10496 <KshitijSaraogi@gmail.com>
+Langston Barrett <langston.barrett@gmail.com> siddharthist <langston.barrett@gmail.com>
+Devang Kulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in> devangkulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
+Mohammad Sadeq Dousti <msdousti@gmail.com> msdousti <msdousti@gmail.com>
+Rohit Rango <rohit.rango@gmail.com> rohitrango <rohit.rango@gmail.com>
+Sergey Pestov <pestov-sa@yandex.ru> psqq <pestov-sa@yandex.ru>
+Varun Garg <varun.garg03@gmail.com> mamakancha <varun.garg03@gmail.com>
+Rajat Thakur <rajatthakur1997@gmail.com> rajatthakur <rajatthakur1997@gmail.com>
+Tanay Agrawal <tanay_agrawal@hotmail.com> tanayag <tanay_agrawal@hotmail.com>
+Mohit Chandra <mohit.chandra@research.iiit.ac.in> mohit3011 <mohit.chandra@research.iiit.ac.in>
+Amit Kumar <dtu.amit@gmail.com> AMiT Kumar <aktech@users.noreply.github.com>
+Amit Kumar <dtu.amit@gmail.com> AMiT Kumar <dtu.amit@gmail.com>
+Harshil Goel <harshil158@gmail.com> Harshil Goel <darkcoderrises@users.noreply.github.com>
+Ruslan Pisarev <rpisarev@cloudlinux.com> Ruslan Pisarev <ruslan@rpisarev.org.ua>
+Langston Barrett <langston.barrett@gmail.com> siddharthist <langston.barrett@gmail.com>
+Isuru Fernando <isuruf@gmail.com> Isuru Fernando <isuru.11@cse.mrt.ac.lk>
+Arif Ahmed <arif.ahmed.5.10.1995@gmail.com> ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
+Valeriia Gladkova <valeriia.gladkova@gmail.com> Val <valeriia.gladkova@gmail.com>
+Valeriia Gladkova <valeriia.gladkova@gmail.com> valglad <valeriia.gladkova@gmail.com>
```

Comment:
Every variation in the git history has an entry in the mailmap file. The version in AUTHORS is what I want you to confirm is what you want.